docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+<p class="blog-description">1200 out of 1589 answered</p>
+
+Developers are mostly interested in React design systems, codebase generators, tools for internal needs, data visualization, and prototyping. These needs are both for internal and customer-facing use-cases.
+
+## About you
+
+### How did you first hear about MUI?
+
+<img src="/static/blog/2021-survey/30.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Organic search: 57.8%; Word of mouth: 25.8%; Social media: 8.4%; Blog: 4.4%; Other: 3.5%" />
+
+<p class="blog-description">1417 out of 1589 answered</p>
+
+### Which of the following best describes your current job role?
+
+<img src="/static/blog/2021-survey/31.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Full-stack developer: 52.6%; Front-end developer: 30.5%; Entrepreneur (I do it all!): 7.5%; Beginner learning web development: 2.6%; Engineering Manager: 2.6%; Back-end developer: 1.1%; Designer: 1%; Product Manager: 0.7%; Other: 1.3%" />
+
+<p class="blog-description">1497 out of 1589 answered</p>
+
+### How many total employees work at your current place of employment?
+
+<img src="/static/blog/2021-survey/32.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="0-1: 187; 2-10: 358; 11-20: 136; 21-50: 139; 51-100: 97; 100+: 305" />
+
+<p class="blog-description">1222 out of 1589 answered</p>
+
+### How long have you been developing with JavaScript?
+
+<img src="/static/blog/2021-survey/33.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Just getting started: 4.8%; 1 year +: 22.2%; 3 years +: 27.9%; 5 years +: 28.2%; 10 years +: 11.1%; 15 years +: 5.9%" />
+
+<p class="blog-description">1520 out of 1589 answered</p>
+
+### How long have you been developing with React?
+
+<img src="/static/blog/2021-survey/34.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Just getting started: 7%; 6 months +: 11%; 1 year +: 20.6%; 2 years +: 21.2%; 3 years +: 27.1%; 5 years +: 11.5%; Bleeding edge baby! 6 years +: 1.6%" />
+
+<p class="blog-description">1523 out of 1589 answered</p>
+
+### How long have you been developing with MUI?
+
+<img src="/static/blog/2021-survey/35.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Just getting started: 16.6%; 6 months +: 19.4%; 1 year +: 24.4%; 2 years +: 21.2%; 3 years +: 15.6%; I'm a pioneer! 5 years +: 2.8%" />
+
+<p class="blog-description">1519 out of 1589 answered</p>
+
+## Conclusion
+
+The annual survey is one of the most important resources we have for deciding on the MUI next steps. We appreciate a lot the time you invested in answering the questions and we can't say enough how invaluable this data is for our team.
```

Comment:
Rather than framing it in terms of how great it is for us, we should focus on how much we appreciate our super fans.

```suggestion
The annual survey is one of the most important resources we have for deciding on our next steps. We appreciate all the time you invested in answering our questions, and we can't stress enough how much we value your input.
```